CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-# v0.3.0
+# v0.4.0
```

Comment:
It seems like new clients will be confused by trying to talk to old servers, with this protocol change. Do you want to add a warning to restart them? Or even some inline marker in the connection?

Reply:
Added